modules/order/order.graphql.js

```diff
@@ -63,6 +63,7 @@ type OrderUser {
 `;
 const orderInputs = ` 
 input OrderInput {
+  _id:ID
```

Comment:
Why do we need to input manually id field?

Reply:
Because i do not want to write separate function on front to strip the id from the fetched object. Id cannot be changed anyway